src/hooks/useSolanaConnection.ts

```diff
@@ -0,0 +1,10 @@
+import { Connection as SolanaConnection } from '@solana/web3.js';
+
+import { useEndpointsConfig } from '@/hooks/useEndpointsConfig';
+
+// Create and export a single instance of Connection
```

Comment:
[q] wouldn't this create a new connection every time it's used unless we save it in state or a memo?

Reply:
good point, i'll wrap in in a useMemo so we only create one per `endpointsConfig.solanaRpcUrl`

Reply:
✅